src/Bridge/ExceptionListener.php

```diff
@@ -0,0 +1,74 @@
+<?php
+declare(strict_types=1);
+
+namespace Jaeger\Symfony\Bridge;
+
+use Jaeger\Span\SpanManagerInterface;
+use Jaeger\Tag\ErrorTag;
+use Jaeger\Tracer\TracerInterface;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\Event\ExceptionEvent;
+use Symfony\Component\HttpKernel\Event\RequestEvent;
+use Symfony\Component\HttpKernel\Event\TerminateEvent;
+use Symfony\Component\HttpKernel\HttpKernelInterface;
+
+class ExceptionListener implements EventSubscriberInterface
+{
+    private $tracer;
+    private $spanManager;
+    private $globalSpanHandler;
+    private $exceptionExist;
+
+    public function __construct(
+        TracerInterface $tracer,
+        SpanManagerInterface $spanManager,
```

Comment:
Don't ask more than you need, here the SpanAwareInterface is sufficient for your needs instead of SpanManagerInterface

Reply:
Fixed